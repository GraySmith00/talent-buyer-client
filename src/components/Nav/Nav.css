.frequency {
  width: 2rem; }

.logo-wrap {
  align-items: center;
  display: flex;
  justify-content: space-around;
  width: 80%;
  padding-left: 2rem; }

.nav {
  background-color: #131621;
  color: #fff;
  min-width: 15rem;
  height: 100vh; }
  .nav .active {
    border-left: solid 5px #D52F7F;
    background-color: #1e2334;
    cursor: pointer;
    color: #D52F7F; }

.nav-link-wrapper {
  display: flex;
  flex-direction: column; }

.logo {
  color: #fff;
  font-family: "PT Sans", sans-serif; }

.nav-link {
  font-family: 'Open Sans', sans-serif;
  font-weight: 700;
  width: 100%;
  padding: 1.5rem 0 1.5rem 2rem;
  border-left: solid 5px #131621;
  color: #5d5d61;
  font-size: 1.5rem;
  text-decoration: none;
  outline: none; }
  .nav-link:hover {
    border-left: solid 5px #D52F7F;
    background-color: #1e2334;
    cursor: pointer;
    color: #D52F7F; }

.venue-info {
  display: flex;
  flex-direction: column;
  align-items: center;
  margin: 2rem auto;
  width: 80%;
  background-color: rgba(250, 250, 250, 0.1);
  height: 300px;
  border-radius: 6px;
  color: #fff; }
  .venue-info header {
    font-family: 'Open Sans', sans-serif;
    font-weight: 700;
    margin: 0 auto;
    padding: 1rem;
    width: 100%;
    font-size: 1.3rem;
    background-color: #5a668a;
    border-radius: 6px 6px 0px 0px;
    -moz-border-radius: 6px 6px 0px 0px;
    -webkit-border-radius: 6px 6px 0px 0px;
    border: 0px solid #000000; }
  .venue-info .venue-info-main {
    display: flex;
    flex-direction: column;
    justify-content: space-around; }
