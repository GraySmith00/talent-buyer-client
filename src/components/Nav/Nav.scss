.nav {
  background-color: #131621;
  color: #fff;
  width: 15rem;
  height: 100vh;
}
