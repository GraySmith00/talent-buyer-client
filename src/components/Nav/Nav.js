import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';
import './Nav.css';

class Nav extends Component {
  render() {
    return (
      <div className="nav">
        <div className="logo-wrap">
          <img
            className="frequency"
            src={require('./frequency.svg')}
            alt="frequency-logo"
          />
          <h1 className="logo">TalentBC</h1>
        </div>
        <section className="nav-link-wrapper">
          <NavLink to="/home" className="nav-link">
            Home
          </NavLink>

          <NavLink to="/artists" className="nav-link">
            Artists
          </NavLink>
        </section>
      </div>
    );
  }
}

export default Nav;
