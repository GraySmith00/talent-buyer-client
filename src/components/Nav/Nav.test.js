import React from 'react';
import { shallow } from 'enzyme';

import Nav from './Nav';

describe('Nav component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Nav />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
