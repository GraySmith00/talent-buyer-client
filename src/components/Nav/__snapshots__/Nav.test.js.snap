// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Nav />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h2>
        Nav
</h2>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Nav",
      },
      "ref": null,
      "rendered": "Nav",
      "type": "h2",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h2>
          Nav
</h2>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Nav",
        },
        "ref": null,
        "rendered": "Nav",
        "type": "h2",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
